### ISSUE TITLE
Not match the pretrained ckpt

### ISSUE THREAD

**Issue by hyysam** [ruizhecao96/CMGAN#12]
I use VCTK-DEMAND to train the model (120 epoch)，but when I use the 120 epoch ckpt to inference the VCTK-DEMAND noisy data，the denoising effect is not  as good as the pretrained ckpt.
Did you use extract dataset to train the model?

**Comment by hyysam** [comment]
Can you show me your training loss tensorboard?

**Comment by ruizhecao96** [comment]
> Can you show me your training loss tensorboard?

Sorry I didn't record the loss on tensorboard, but we have use the early stop method since it gets overfitting when epoch number is too large, usually 80 epochs would be enough.

**Comment by hyysam** [comment]
I find that the --loss_weights", default=[0.1, 0.9, 0.2, 0.05] is not same as the paper default, and let loss_gan be smaller than the other loss, finally the GAN is not  work

**Comment by hyysam** [comment]
Do you use extract dataset to train the pretrained ckpt?

**Comment by ruizhecao96** [comment]
Since the GAN loss is much large than other losses, we add the weights to scale them to the same level. We didn't use any other dataset to train the model.

**Comment by hyysam** [comment]
Thanks your answer
